src/Application/ApprovalState.php

```diff
@@ -8,12 +8,14 @@ class ApprovalState {
 
 	/**
 	 * @param int|null $approverId ID of the user who (un)approved the page, or null for automatic unapproval
+	 * @param string|null $approverUserName
```

Comment:
This is unnecessary. The line above is there because of the description, not for typehinting.